src/imitation/data/huggingface_utils.py

```diff
@@ -124,6 +124,8 @@ def trajectories_to_dict(
         ],
         terminal=[traj.terminal for traj in trajectories],
     )
+    if any(isinstance(traj.obs, types.DictObs) for traj in trajectories):
```

Comment:
I do intend to support this.

Reply:
Yes, we should support this eventually, although given time constraints (and size of this PR) I'd be fine leaving it undone and just opening an issue to track adding support for this eventually.

Reply:
Will not support in this PR.